docs/configuration/monitoring.md

```diff
@@ -58,6 +58,17 @@ Crow is a visualization tool that displays information about the projects and jo
 
 The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
 
+The `crow` tool can be accessed through the `/monitoring` path of the url pointing to your Codacy installation, e.g. `http://codacy.company.org/monitoring`.
+You must set the `crow.config.codacy.url` and `crow.config.crow.url` values in your `values.yaml` file so that anchor links to your projects can be properly established inside `crow`.
+
+You must also provide a `CROW_PASSWORD` environment variable for your Crow password:
```

Comment:
I don't think this part makes sense here. This is a good section to be in the readme of crow tought

Reply:
But how are we going to explain on the self-hosted docs that the customer needs to configure that variable to have the credentials configured?